admin/README.md

```diff
@@ -1,48 +1,14 @@
 # Admin workflow
-To kick off a conference, the OpenReview superuser needs to:
+To kick off a conference, the OpenReview superuser should:
 - Create a conference directory under openreview-scripts/venues
-- Create and post the conference's root group to the system
-- Create the conference administrator account (Optional)
+- Copy the file openreview-scripts/admin/conference-template/params.data and fill it in with conference-specific variables
+- Run openreview-scripts/admin/superuser-init.py (see comments in that file for details)
 
-the `superuser-init.py` script is responsible for performing all three of these tasks. Call it with the following arguments:
 
-```
-optional arguments:
-  -h, --help            show this help message and exit
-  -c CONF, --conf CONF  the full path of the conference group to create
-  --overwrite           if true, overwrites the conference directory
-  --data DATA           a .json file containing parameters. For each parameter
-                        present, the corresponding user prompt will be skipped
-                        and replaced with that value.
-  --baseurl BASEURL     base URL
-  --username USERNAME
-  --password PASSWORD
-```
-
-`superuser-init.py` also generates a file at the top level of the conference directory called `params.json`. This file contains all the parameters that were initially passed into the script's text prompts. Passing the path of `params.json` into the `--data` flag allows the user to skip the text prompts and generate new files using those parameters. When combined with the `--overwrite` flag, this is useful for making changes to the conference configuration.
-
-```
 Example:
 
->> superuser-init.py --conf myconf.org/MYCONF/2017 --data ./params.json 
+```
+>> superuser-init.py --conf myconf.org/MYCONF/2017 --data path/to/params.data
```

Comment:
superuser-init could read the params.data from a default location.

Reply:
what is the conf param? the folder of the conference? 'conf' is confusing with the 'configuration' word. Could we use --venue-path or --venue ?

Reply:
ok, I will make the --data parameter optional and look for it in the venue directory